commands/user.go

```diff
@@ -91,6 +100,10 @@ func init() {
 	UserCreateCmd.Flags().String("locale", "", "Optional. The locale (ex: en, fr) for the new user account.")
 	UserCreateCmd.Flags().Bool("system_admin", false, "Optional. If supplied, the new user will be a system administrator. Defaults to false.")
 
+	ListUsersCmd.Flags().Int("page", 0, "Start page for list of users")
+	ListUsersCmd.Flags().Int("per-page", 200, "Number of users to be fetched")
+	ListUsersCmd.Flags().Bool("all", false, "Fetch all users. Will ignore --page and --per-page")
```

Comment:
I don't see how this ignores `--page` and `--per-page`. From what I understand, if `all` is set, it just increments the page number and keeps on getting new pages. That's the only difference. Can you clarify ?

Also, there is no need for it to ignore `per-page`. It should work fine with all. And that's what it seems to do. Just that it should ignore page, which it does not seem to do.

Reply:
Sounds good. Just a small nit: it should be `--page flag will be ignored if provided`. Thanks !